test/conformance/image/go-runner/cmd.go

```diff
@@ -72,7 +72,9 @@ func getCmd(env Getenver, w io.Writer) *exec.Cmd {
 	if len(env.Getenv(dryRunEnvKey)) > 0 {
 		ginkgoArgs = append(ginkgoArgs, "--dryRun=true")
 	}
-
+	// NOTE: Ginkgo's default timeout has been reduced from 24h to 1h in V2, set it manually here as "24h"
+	// for backward compatibility purpose.
+	ginkgoArgs = append(ginkgoArgs, "--timeout=24h")
```

Comment:
shouldn't this first check if `ginkgoArgs` already contains a --timeout arg (or if there's a short version of the flag as well)?

Reply:
> or if there's a short version of the flag

How about the `ginkgoTimeout`?

Reply:
I read the code again, `ginkgoArgs` don't get any chance to contain `timeout` parameter,  and is that really worth for an extra looping against the slice and checking that slice contains `timeout` parameter?

Reply:
I don't context for how this is used... I just don't want to force a config to use 24 hours if it was previously passing in args to set a different explicit timeout

Reply:
for why this is set to 24 hours, this is the link to the discussion with @pohly @onsi @aojea , it is started from here: https://github.com/kubernetes/kubernetes/pull/109111#discussion_r857169282

mostly is for backward-compatibility, I have observed that there are some long-running testcase will abort too early due to the default timeout changes in `Ginkgo`.

Reply:
this will be updated and UT-ed in the next iteration.